test/cppsuite/test_harness/util/debug_utils.h

```diff
@@ -30,29 +30,71 @@
 #define DEBUG_UTILS_H
 
 #include <iostream>
+#include <sstream>
+#include <thread>
+
+extern "C" {
+#include "wt_internal.h"
+#include "test_util.h"
+}
 
 /* Define helpful functions related to debugging. */
 namespace test_harness {
 
-#define DEBUG_ERROR 0
-#define DEBUG_WARN 1
-#define DEBUG_INFO 2
-#define DEBUG_TRACE 3
+/* Possible log levels. If you change anything here make sure you update LOG_LEVELS accordingly. */
+#define LOG_ERROR 0
+#define LOG_WARN  1
+#define LOG_INFO  2
+#define LOG_TRACE 3
+
+/* Order of elements in this array corresponds to the definitions above. */
+const char * const LOG_LEVELS[] = { "ERROR", "WARN", "INFO", "TRACE" };
+
+/* Current log level */
+static int64_t _trace_level = LOG_WARN;
+
+/* If true both date and time will be logged, if false only time. */
+static bool _include_date = true;
+
+void get_time(char *time_buf, size_t buf_size) {
+
+    size_t alloc_size;
+    struct timespec tsp;
+    struct tm *tm, _tm;
 
-static int64_t _trace_level = DEBUG_WARN;
+    __wt_epoch(NULL, &tsp);
```

Comment:
Can we use <chrono> instead of the wt provided time implementations? That will be more consistent with other parts of the framework. 

Just make sure it compiles on gcc version 4.8.

Reply:
Workgen uses the same way of getting time.

Reply:
If it doesn't compile it should be detected by our evergreen compilers task before PR merge, but its worth a heads up as some C++11 time things only exist in gcc 5.

Reply:
Changed to `<chrono>`. Thanks!